File: SchoolSync_SpringBoot/backend/src/main/java/com/schoolsync/dto/StudentDTO.java
package com.schoolsync.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StudentDTO {
    private Long studentId;
    private String username;
    private String rollNo;
    private String className;
    private String email;
    private String phone;
}

File: SchoolSync in MS.NET/Backend in .NET/DTOs/StudentDto.cs
namespace SmartSchoolAPI.DTOs;

public class StudentDto
{
    public int? StudentId { get; set; }
    public string Username { get; set; } = string.Empty;
    public string Password { get; set; } = string.Empty;
    public string? Email { get; set; }
    public string? Phone { get; set; }
    public string RollNo { get; set; } = string.Empty;
    public string Class { get; set; } = string.Empty;
}

